infra/modules/providers/azure/app-service/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "service_plan_resource_group_name" {
+  description = "The name of the resource group in which the service plan was created."
+  type        = "string"
+}
+
+
+variable "service_plan_name" {
+  description = "The name of the service plan"
+  type        = "string"
+}
+
+variable "resource_tags" {
+description = "Map of tags to apply to taggable resources in this module. By default the taggable resources are tagged with the name defined above and this map is merged in"
+type        = "map"
+default     = {}
+}
+
+variable "app_service_name" {
+  description = "The name of the app service to be created"
+  default     = ["appsvc11", "appsvc12"]
+}
+
+variable "docker_registry_server_url" {
+  description = "The docker registry server URL for app service to be created"
+  default     = ["", ""]
+}
+
+variable "docker_registry_server_username" {
+  description = "The docker registry server username for app service to be created"
+  default     = ["", ""]
+}
+
+variable "docker_registry_server_password" {
+  description = "The docker registry server password for app service to be created"
+  default     = ["", ""]
+}
+
+variable "site_config_linux_fx_version" {
+  description = "The Linux App Framework and version for the App Service to be created"
```

Comment:
I feel like the  app service name and container should be merged as a single variable. Here’s an example for how this was done in a similar scenario https://link.medium.com/BDrofpzBcW

Reply:
I believe this is an improvement task that needs to be added for future sprints.